docs/topics/project-management.md

```diff
@@ -97,9 +97,15 @@ The following template should be used for the description of the issue, and serv
     Release manager is @***.
     Pull request is #***.
 
+    Pre checklist:
```

Comment:
Actually this should be during the development cycle in order to leave enough time to the translators to work on it.
I started discussing with Claude about how they handle that in Django.